resources/js/profile-page/badges.tsx

```diff
@@ -24,6 +24,7 @@ export default class Badges extends React.PureComponent<Props> {
           const img = (
             <Img2x
               className='profile-badges__badge'
+              data-date={badge.awarded_at}
```

Comment:
Format the description with the date here and pass as `data-html-title` instead of adding new functions to `TooltipDefault`